testdata/routing/images/caddy-docker/Dockerfile

```diff
@@ -0,0 +1,19 @@
+# Original caddy-docker build file.
+FROM alpine:3.2
+
+RUN apk add --update openssh-client git tar
+
+ADD caddy /usr/bin/
+
+RUN chmod 0755 /usr/bin/caddy
+
+EXPOSE 8080 8443
+WORKDIR /srv
+
+ADD Caddyfile /etc/caddy/Caddyfile
+ADD public/ /srv/public/
+ADD publics/ /srv/publics/
+ADD certs/ /srv/certs
```

Comment:
For completeness, those files/folders that `ADD` to the image should be upload to the same folder as well and ensure using relative path.

Reply:
Good catch!  I have added all those files to the relative directory in the same tree structure as the original ones. Let me know if there is anything further to be done. 

Thanks.